beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -1102,6 +1109,10 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
             return Ok(BlockProcessingOutcome::GenesisBlock);
         }
 
+        if block.slot >= MAXIMUM_BLOCK_SLOT_NUMBER {
```

Comment:
I believe this is all there is to add. I tried looking for everything related to block_processing, block_importing, import_block, and this is the only place I found some relevant code.